package.json

```diff
@@ -7,6 +7,7 @@
 	},
 	"dependencies": {
 		"firebase": "^9.15.0",
+		"node": "^19.3.0",
```

Comment:
I’m guessing this was added by accident! We don’t want node itself as a dependency in this project, so please consider removing this before merging.

`npm uninstall node` should do the trick!